### DISCUSSION TITLE
Cannot flash the bios to gigabyte mobo

### DISCUSSION THREAD

**Discussion by KiKiHUN1** [xCuri0/ReBarUEFI#146] [General]
Idk what the fk is wrong with my efiflash.
It tells me that it is writing to the chip, it verifies the chip, but in the bios it still on the older version. I don't get it.
Btw. Any help what other program can i use to flash the modded bios? 
I don't have an 1.8v adapter for the bios programmer thing.
Gigabyte B660m ds3h ax

**Comment by ZOXZX** [comment]
If you are using ch341 then it is a simple mod (one wire) to have 1.8V.
